sevsnpmeasure/guest.py

```diff
@@ -18,16 +19,23 @@
 
 def calc_launch_digest(mode: SevMode, vcpus: int, vcpu_sig: int, ovmf_file: str,
                        kernel: str, initrd: str, append: str, snp_ovmf_hash_str: str = '',
-                       vmm_type: VMMType = VMMType.QEMU) -> bytes:
+                       vmm_type: VMMType = VMMType.QEMU, dump_vmsa: bool = False, svsm_file: str = '',
+                       ovmf_vars_size: int = 0) -> bytes:
```

Comment:
at some point it would make sense to create a LaunchDigestOptions class with all these arguments, but for this PR you can leave it like this.

Reply:
I agree.

Does it still make sense to have the common `calc_launch_digest()` or just offer the individual functions, one for each mode?

Reply:
Good question. Maybe both - like the requests library has both a generic `request()` function and individual `head()`, `get()`, `post()` functions. Even then, most of our individual mode functions take more than 5 arguments -- it might make sense to pack them in a class, or use kwargs, like: https://requests.readthedocs.io/en/latest/api/#requests.request 

Something to think about.